opencodelists/models.py

```diff
@@ -110,12 +110,28 @@ def is_admin_member(self, organisation):
 
     @property
     def codelists(self):
-        return Codelist.objects.filter(handles__user=self)
+        """Codelists owned by user"""
+        return Codelist.objects.filter(handles__is_current=True, handles__user=self)
 
     @property
     def drafts(self):
+        """Codelist versions authored by user and in draft"""
         return self.versions.filter(status=Status.DRAFT)
 
+    @property
+    def reviews(self):
```

Comment:
I think this is a slightly misleading name, and I'd prefer `versions_under_review`.  This is because we might introduce the concept of something like a code review in the future.